Actions/CalculateArtifactNames/CalculateArtifactNames.ps1

```diff
@@ -19,8 +19,13 @@ function Set-EnvVariable([string] $name, [string] $value) {
 
 $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
 
-Write-Host $settingsJson
-$settings = $settingsJson | ConvertFrom-Json
+# Support potentially base64 encoded settings
+try {
```

Comment:
This construct seems off. Shouldn't the action's contract specify if the settings are base64-encoded or not?

Reply:
Due to the nature of our preview - we will have old versions of AL-Go workflows calling into this function with a json construct which isn't base64 - and every new version will be base64.
I assume that once we have shipped this - we will be able to remove the support for non-base64 in preview - but for now, we have to support both in order to not break partners.

I could also have added the useBase64 parameter - but that one (as this construct) is temporary and can be removed later.